dnsmasq-metrics/build/Dockerfile.e2e

```diff
@@ -0,0 +1,10 @@
+# Builds the docker container for the e2e test.
```

Comment:
I would put all the e2e stuff in a dir `test` or `e2e` or something, rather than `build`